var/spack/repos/builtin/packages/py-timm/package.py

```diff
@@ -32,15 +33,15 @@ class PyTimm(PythonPackage):
     with default_args(type=("build", "run")):
         # https://github.com/huggingface/pytorch-image-models/issues/1530
         # https://github.com/huggingface/pytorch-image-models/pull/1649
-        depends_on("python@:3.10", when="@:0.6.12", type=("build", "run"))
+        depends_on("python@:3.10", when="@:0.6.12")
 
-        depends_on("py-torch@1.7:", when="@0.6:", type=("build", "run"))
-        depends_on("py-torch@1.4:", type=("build", "run"))
-        depends_on("py-torchvision", type=("build", "run"))
-        depends_on("py-pyyaml", when="@0.6:", type=("build", "run"))
-        depends_on("py-huggingface-hub", when="@0.6:", type=("build", "run"))
-        depends_on("py-safetensors", when="@0.9:", type=("build", "run"))
+        depends_on("py-torch@1.7:", when="@0.6:")
```

Comment:
FWIW. Not getting this version from `pyproject.toml`.

Reply:
It was documented in timm 0.X back when they used a setup.py but removed when they switched to pyproject.toml in timm 1.X. PyTorch 1.6 and older are deprecated anyway, so I don't think it matters too much whether we keep this or remove it.

Reply:
My only reservation would be if someone or a facility has older external installations of `py-torch` that could get picked up without the lower limit.